Formula/dcmtk.rb

```diff
@@ -17,20 +17,23 @@ class Dcmtk < Formula
 
   option "with-docs", "Install development libraries/headers and HTML docs"
   option "with-libiconv", "Build with brewed libiconv. Dcmtk and system libiconv can have problems with utf8."
+  option "with-dicomdict", "Build with baked-in DICOM data dictionary."
+  option "without-openssl", "Build without support for OpenSSL."
```

Comment:
Thanks again for the detailed explanation. I don't see a reason to expect them to change the default, and in any case if that happens we can always adapt the formula accordingly.